File: sdk/agid-r/R/agid.R
BASE32_ALPHABET <- "0123456789ABCDEFGHJKMNPQRSTVWXYZ"
AGID_PREFIX_LENGTH <- 2
AGID_HASH_LENGTH <- 10
AGID_TOTAL_LENGTH <- 12

agid_encode <- function(lat, lon) {
  stop("wire this package to the AGID reference implementation")
}

agid_decode <- function(id) {
  NULL
}

agid_cellBounds <- function(id) {
  stop("wire this package to the AGID reference implementation")
}

File: sdk/agid-c/src/agid.c
#include "agid.h"

int agid_encode(double lat, double lon, agid_result* out) {
  (void)lat;
  (void)lon;
  (void)out;
  return -1;
}

int agid_decode(const char* id, agid_result* out) {
  (void)id;
  (void)out;
  return -1;
}

int agid_cell_bounds(const char* id, agid_bounds* out) {
  (void)id;
  (void)out;
  return -1;
}
